magma_access_gateway_installer/__init__.py

```diff
@@ -17,78 +16,75 @@
 from .agw_installation_service_creator import AGWInstallerInstallationServiceCreator
 from .agw_installer import AGWInstaller
 from .agw_network_configurator import AGWInstallerNetworkConfigurator
-from .agw_preinstall_checks import AGWInstallerPreinstallChecks
+from .agw_preinstall import AGWInstallerPreinstall
 from .agw_service_user_creator import AGWInstallerServiceUserCreator
 
 logger = logging.getLogger(__name__)
-handler = JournalHandler()
-handler.setFormatter(logging.Formatter("Magma AGW Installer: %(message)s"))
-logger.addHandler(handler)
 logger.setLevel(logging.INFO)
-logger.addHandler(logging.StreamHandler(sys.stdout))
+journal_handler = JournalHandler()
+journal_handler.setFormatter(logging.Formatter("Magma AGW Installer: %(message)s"))
+logger.addHandler(journal_handler)
+stdout_handler = logging.StreamHandler(sys.stdout)
+stdout_handler.setFormatter(logging.Formatter("Magma AGW Installer: %(message)s"))
```

Comment:
Do we need the "Magma AGW Installer" section? when you run `magma-access-gateway.install` you already know you're in the context of installing magma.

Reply:
We don't need it, but since there's gonna be a lot of noise from apt (can't really make it quiet), I thought it's gonna be easier to spot messages produced by the installer if they all get common prefix.

Reply:
Ok good